data/json/items/tool/electronics.json

```diff
@@ -195,7 +172,6 @@
     "use_action": [
       "EINKTABLETPC",
       "ELECTRICSTORAGE",
-      "EBOOKSAVE",
```

Comment:
Was this intended?

Reply:
Yes, eink readers do not have cameras, therefore should not have any capability to scan books

Reply:
Well, some models have cameras and even OCR software. I own one of those - PocketBook 650 (and I am pretty sure there are other models with cameras too).

Reply:
huh, i honestly failed to find such
if this exist, i think item can be revisited to give ebooksave back (and check it's power consumption, some books are too big for it to be scanned, which i think is incorrect)

Reply:
While that's a neat device, I don't think it's a good general representation of e-ink tablets that you're likely to find in people's homes in the US.

Reply:
@harakka Do you refer to the ones with camera as being unlikely?

Reply:
Yes.